File: WebApp/WebApp/Models/Flight.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace WebApp.Models
{
    public class Flight
    {
        public int Id { get; set; }
        public string Origin { get; set; }
        public string Destination { get; set; }
        public DateTime TakeoffTime { get; set; }
        public DateTime LandingTime { get; set; }
        public string Duration { get; set; }
        public string Distance { get; set; }
        public string Stops { get; set; }
        public string Price { get; set; }
        public int SeatsLeft { get; set; }
    }
}

File: angular-app/src/models/Flight.ts
import { Airline } from './Airline';

export class Flight {
    public constructor(init?: Partial<Flight>) {
        Object.assign(this, init);
    }

    Id: number;
    Origin: string;
    Destination: string;
    TakeoffTime: Date;
    LandingTime: Date;
    Duration: string;
    Distance: string;
    Stops: string;
    Price: string;
    SeatsLeft: number;
    Airline: Airline;
}